### PR TITLE
add a nag to list all recent dups

### PR DESCRIPTION
**Pull request by sylvestre**

### CODE REVIEW

File: auto_nag/scripts/several_dups.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+
+class SeveralDups(BzCleaner):
+    def __init__(self):
+        super(SeveralDups, self).__init__()
+        self.nweeks = utils.get_config(self.name(), "weeks_lookup")
+        self.ndups = utils.get_config(self.name(), "number_dups")
+
+    def description(self):
+        return 'Defect with several dups for the last {} weeks'.format(
```
Comment (marco-c) [inline auto_nag/scripts/several_dups.py:16]: You are not filtering on defects, did you mean to?
If not, s/Defect/Bugs

**Comment by marco-c** [comment]
We can build #1184 on top of this script

**Comment by marco-c** [comment]
If we don't want to restrict to defects, maybe we should at least skip meta bugs.

And maybe we should skip bugs about intermittent tests, wdyt?

**Comment by sylvestre** [comment]
good ideas, fixed

File: auto_nag/scripts/configs/tools.json [hunk 2]
```diff
@@ -229,6 +229,11 @@
     {
         "months_lookup": 24
     },
+    "several_dups":
+    {
+        "weeks_lookup": 3,
+        "number_dups": 3
+    },
```
Comment (marco-c) [inline auto_nag/scripts/configs/tools.json:236]: @calixteman do we need to add a `receivers` with `release-mgmt@mozilla.com`, or is it default?

**Review by marco-c — DISMISSED** [review]
lgtm

File: auto_nag/scripts/configs/tools.json (hunk 2 above)
Comment (calixteman) [reply auto_nag/scripts/configs/tools.json:236]: yes we do.

**Review by calixteman — APPROVED** [review]
LGTM, thank you

**Pull request merged**